frontend/libs/studio-components/src/components/StudioPageHeader/StudioPageHeaderLink/StudioPageHeaderLink.tsx

```diff
@@ -0,0 +1,31 @@
+import type { HTMLAttributes, ReactElement } from 'react';
+import commonClasses from '../common.module.css';
+import { type StudioPageHeaderColor } from '../types/StudioPageHeaderColor';
+import cn from 'classnames';
+import { type StudioPageHeaderVariant } from '../types/StudioPageHeaderVariant';
+import linkClasses from './StudioPageHeaderLink.module.css';
+
+export type StudioPageHeaderLinkProps = {
+  color: StudioPageHeaderColor;
+  variant: StudioPageHeaderVariant;
+  renderLink: (props: HTMLAttributes<HTMLAnchorElement>) => ReactElement;
+} & HTMLAttributes<HTMLAnchorElement>;
+
+export function StudioPageHeaderLink({
```

Comment:
You have probably already thought about it, but is it not more precisely calling this component `StudioPageHeaderHeaderLink`? But I see that its a mouthfull 😅

Reply:
I agree. I didn't change it because it would require changing `StudioPageHeader` as well, but I've made changes there anyways. I'll fix.